content/_data/changelogs/weekly.yml

```diff
@@ -7034,6 +7034,89 @@
   # pull: 4596 (PR title: [JENKINS-61409] Websockets: Use AbstractByteBufferCommandTransport to transport messages) combined with pull 4601
   # pull: 4605 (PR title: Do not print random binary data during WebSocketAgentsTest)
 
+- version: 2.230
+  date: 2020-04-04
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4610
+    issue: 61478
+    authors:
+    - romenrg
+    message: |-
+      Style improvements on alert banners.
```

Comment:
Add some details so that users can understand what exactly changed? Also, entry should be something like "Improve style of alert banners" to follow the changelog style

Reply:
Fixed in 548a03e610f2ed957605c81445825746baf7f3d7